com.unity.render-pipelines.universal/Runtime/VFXGraph/Shaders/VFXCommon.hlsl

```diff
@@ -116,14 +116,25 @@ float3 GetWorldStereoOffset()
 
 float VFXSampleDepth(float4 posSS)
 {
-    return LoadSceneDepth(uint2(posSS.xy));
+    float2 screenUV = GetNormalizedScreenSpaceUV(posSS.xy);
+
+    //In URP, the depth texture is optionnal and could be 4x4 white texture, Load isn't approriate in that case.
```

Comment:
```suggestion
    // In URP, the depth texture is optional and could be 4x4 white texture, Load isn't appropriate in that case.
```

Reply:
Thanks, I should enable automatic correction in hlsl file as well.